src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -488,6 +482,29 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
     }
   }
 
+  function commitCallbacks(updateQueue, context) {
```

Comment:
Same thing here. I'd like to keep as the JS object dependent stuff out of the core modules.

Reply:
By JS object dependent stuff, do you mean callbacks since they're kinda unique to JS?